2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -62,9 +62,9 @@ Al fine di eliminare le ambiguità durante la consultazione dei documenti, verr
 = Processi di accordo
 == Processo di fornitura
```

Comment:
Aggiungere dopo il titolo: _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.1.2_ (vedi ad esempio Processo di gestione dei modelli di ciclo di vita, riga 238)